content_sync/pipelines/concourse_test.py

```diff
@@ -628,11 +630,12 @@ def test_upsert_pipeline(
 @pytest.mark.parametrize("prefix", ["", "/test_prefix", "test_prefix"])
```

Comment:
Might be able to get rid of `/test_prefix` as a parameter value if the slash doesn't matter anymore.  Or get rid of the test parameter altogether.

Reply:
This should still be tested, but should likely be moved to where the formatting is actually applied in `SitePipelineDefinitionConfig`

Reply:
This is already being tested in `site_pipeline_test.py`, so I went ahead and took it out of the test in `concourse.py`: https://github.com/mitodl/ocw-studio/blob/31d668b1be930b434c3339e7fe8be0450789fb22/content_sync/pipelines/definitions/concourse/site_pipeline_test.py#L86